src/Storages/ExternalStream/Kafka/WriteBufferFromKafka.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <atomic>
+#include <librdkafka/rdkafka.h>
```

Comment:
It will be good to have a single place to depend on librdkafka headers. So i am expecting most of the utils / functions will go to `KafkaLog` folder and others depend on `KafkaLog`